tekton/tasks/cert-manager.yml

```diff
@@ -48,6 +56,11 @@ spec:
         value: $(params.devops_suite_name)
       - name: DEVOPS_BUILD_NUMBER
         value: $(params.devops_build_number)
+      # Optional parameters to categoriz result to be persisted
```

Comment:
Small typo in various places to find/replace fix `categoriz` -> `categorize`

Reply:
Done